index.js

```diff
@@ -1,11 +1,7 @@
 'use strict';
 
 var assign = require('object.assign');
-
-var modules = [require('babel-plugin-transform-es2015-modules-commonjs'), {
-  strictMode: true, // add "use strict"
-  strict: false // this allows __esModule to be exported
-}];
+var declare = require('@babel/helper-plugin-utils').declare;
```

Comment:
let's change all these `var`s to `const`s, since we dropped node < 6; also, can this be `const { declare } = require('@babel/helper-plugin-utils');`?

Reply:
Great, I've updated the `.eslintrc`. I also changed the `tests-only` script to actually attempt to run `index.js` so we know whether it'll actually work on the current node version.

Reply:
Should `engines` for `node` `>=6` also be set?